### PR TITLE
Testing with Page object model (POM)

### PR DESCRIPTION
**Pull request by mafeZapata**

### CODE REVIEW

**Comment by mafeZapata** [comment]
Capture testing with POM

File: cypress/e2e/buy-shirt.cy.ts [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+  MenuContentPage,
+  PaymentStepPage,
+  ProductsListPage,
+  ShippingStepPage,
+  ShopingCartPage,
+} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productsListPage = new ProductsListPage();
+const shopingCartPage = new ShopingCartPage();
```
Comment (manuelq12) [inline cypress/e2e/buy-shirt.cy.ts]: Typo, i know its placed like this on the notes but its a typo-
Shopping*

File: cypress/e2e/buy-shirt.cy.ts [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+    shopingCartPage.goToCartPage();
+    shopingCartPage.goToCheckoutPage();
+    loginPage.login("aperdomobo@gmail.com", "WorkshopProtractor");
+    addressStepPage.goToShippingStep();
+    shippingStepPage.acceptTermsAndConditions();
+    shippingStepPage.goToPaymentStep();
+    paymentStepPage.selectPaymentMethod();
+    paymentStepPage.confirmOrder();
+
+    paymentStepPage.getOrderLabel().should("have.text", "Your order on My Store is complete."); ;
```
Comment (manuelq12) [inline cypress/e2e/buy-shirt.cy.ts:35]: I see you have the same changes as Salome haha

Comment (mafeZapata) [reply cypress/e2e/buy-shirt.cy.ts:35]: Sorry but could be as a coincidence

Comment (manuelq12) [reply cypress/e2e/buy-shirt.cy.ts:35]: Its fine, i just think its funny

**Review by manuelq12 — APPROVED** [review]

**Comment by manuelq12** [comment]
Lets wait for the page to respond again

**Pull request merged**